fonts.css

```diff
@@ -4,7 +4,7 @@
     font-weight: normal;
     font-display: swap;
 }
-
+/* 
```

Comment:
Why is this being commented out?

Reply:
Hi @Nikhil-Ladha,

This was related to some LHCI testing and attempting to reduce unnecessary processing.

It seemed that only one of the fonts were being referenced, and the other font variants were not being used on the site, which then seemed to be unnecessary declarations (but [perhaps it does not matter](https://web.dev/font-best-practices/)?)

Thanks for asking these questions, I should be adding more detailed comments in the future.

Cheers,
Randy

Reply:
IIRC, some (or most) of the fonts were used, but if you are sure that they are not used anywhere in the site then I am fine with commenting them out for perf improvement.